paper-fab.html

```diff
@@ -147,7 +155,32 @@
           type: Boolean,
           value: false,
           reflectToAttribute: true
+        },
+        
+        /**
+         * The label displayed in the badge. The label is centered, and ideally
+         * should have very few characters.
+         */
+        label: {
+          type: String,
+          observer: '_labelChanged'
+        },
+        
+        /**
+         * This is true if an icon has been set.
+         */
+        isIconFab: {
```

Comment:
This doesn't need to be a public property. I would recommend just using the computed method directly:
```hidden$="{{!_computeIsIconFab(icon, src)}}"```